anchors/service.go

```diff
@@ -45,32 +56,38 @@ func (s *service) GetDocumentRootOf(anchorID AnchorID) (docRoot DocumentRoot, er
 }
 
 // PreCommitAnchor will call the transaction PreCommit on the smart contract
-func (s *service) PreCommitAnchor(ctx context.Context, anchorID AnchorID, signingRoot DocumentRoot, centID identity.DID, signature []byte, expirationBlock *big.Int) (confirmations <-chan *WatchPreCommit, err error) {
+func (s *service) PreCommitAnchor(ctx context.Context, anchorID AnchorID, signingRoot DocumentRoot) (confirmations chan bool, err error) {
+	did, err := getDID(ctx)
+	if err != nil {
+		return nil, err
+	}
+
 	tc, err := contextutil.Account(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	ethRepositoryContract := s.anchorRepositoryContract
-	opts, err := ethereum.GetClient().GetTxOpts(tc.GetEthereumDefaultAccountName())
+	txID := contextutil.TX(ctx)
+
```

Comment:
fix the spacing here

Reply:
this is just my preference, as its logical separation from the code below